ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/router-bgp-vrf-address-families.md

```diff
@@ -172,12 +172,12 @@ router bgp 65001
          neighbor aa::2 rcf out VRF_AFIPV6_RCF_OUT()
          network aa::/64
          no bgp redistribute-internal
-      redistribute connected rcf VRF_AFIPV6_RCF_CONNECTED()
-      redistribute isis include leaked
-      redistribute ospfv3 match external
-      redistribute ospfv3 match internal include leaked
-      redistribute ospfv3 match nssa-external
-      redistribute static route-map VRF_AFIPV6_RM_STATIC
+         redistribute connected rcf VRF_AFIPV6_RCF_CONNECTED()
+         redistribute isis include leaked
+         redistribute ospfv3 match internal include leaked
+         redistribute ospfv3 match external
+         redistribute ospfv3 match nssa-external
+         redistribute static route-map VRF_AFIPV6_RM_STATIC
```

Comment:
Nasty bug you found there. I would appreciate if you could put this in a separate fix PR. Thanks.

Reply:
Raised PR 4552